packages/client/src/views/OfficeHome/OfficeHome.test.tsx

```diff
@@ -32,7 +32,7 @@ import { merge } from 'lodash'
 import * as React from 'react'
 
 import { waitFor, waitForElement } from '@client/tests/wait-for-element'
-import { SELECTOR_ID } from './tabs/inProgress/inProgressTab'
+import { SELECTOR_ID } from './tabs/inProgress/InProgress'
```

Comment:
Please refactor out the "tabs" directory.  The. name "tabs" is confusing.  Just move all files above and let imports update